src/Text/Pandoc/Readers/Org/Blocks.hs

```diff
@@ -634,25 +634,29 @@ orgTable = try $ do
 
   let caption = fromMaybe mempty (blockAttrCaption blockAttrs)
   let orgTbl = normalizeTable <$> rowsToTable rows
+  let attr = case blockAttrKeyValues blockAttrs of
+        [] -> nullAttr
+        _ -> (fromMaybe mempty (blockAttrLabel blockAttrs), [], blockAttrKeyValues blockAttrs)
```

Comment:
Something I didn't consider when I first wrote this: if `blockAttrKeyValues blockAttrs` is `[]` but the `blockAttrs` has a non-empty `blockAttrLabel` property, should we have `attr` as `(fromMaybe mempty (blockAttrLabel blockAttrs), [], [])`?

Reply:
I think the answer is yes.

Reply:
Actually, I guess it's not so simple since having no `blockAttrName` and a non-null `blockAttrLabel` can produce a duplicate `id` in the HTML because of the div-wrapping performed in this function. I'll need to look at this again tomorrow evening.